_data/sites.json

```
@@ -4199,6 +4199,17 @@
         ]
     },
 
+    {
+        "name": "Mail.ru",
+        "url": "https://account.mail.ru/user/delete",
+        "difficulty": "impossible",
+        "notes": "You can deactivate your account and delete all data, but not entirely delete the account itself.",
```

Bringing @Nightfirecat 's comment on issue #374 over here:

> It might be worth adding a link to that support document as a reference for this note.

```suggestion
        "notes": "You can deactivate your account and delete all data, but not entirely delete the account itself (<a href=\"https://help.mail.ru/mail-help/faq/delete">source</a>).",
```